eng/pipelines/templates/steps/use-venv.yml

```diff
@@ -8,8 +8,27 @@ parameters:
   - name: Condition
     type: string
     default: succeeded()
+  - name: UseUv
```

Comment:
We don't currently set `UseUv`. So we can change this to default to `false` at any time and use bog-standard `pip` if we need to.